### ISSUE TITLE
Update ransack to allow Rails 7

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/openfoodnetwork#10386]
### What we should change and why (this is tech debt)

The upgrade to Rails 7 is blocked by our old ransack version. We forgot to remove the version restriction 2 years ago and haven't received any updates since.

Updating to version 2.5.0 should be enough to support Rails 7. An upgrade to 2.6.0 shouldn't be a problem either. The upgrade to ransack 3 will require a lot of code changes and should be done in a separate pull request. It shouldn't block Rails 7.

https://github.com/activerecord-hackery/ransack/blob/main/CHANGELOG.md#250---2021-12-26

### Context

https://github.com/mkllnk/openfoodnetwork/actions/runs/4100955666/jobs/7072270813

### Impact and timeline

We would like to upgrade to Rails 7 before support for Rails 6.1 runs out.

**Comment by lauriejefferson** [comment]
I'm new to this project and would like to work on this issue.  Where should I start?

**Comment by mkllnk** [comment]
I think that this issue is out of date. There's a reference to a pull request that updated ransack and we are running Rails 7 already. I'm sorry, I just forgot to close this issue. I'll do that now. Is there another task you would like to work on?
